### PR TITLE
clarify we do sticky bucketing

### PR DESCRIPTION
**Pull request by mattdp**
### Features and Changes

- Closes **(add link to issue here)**

### CODE REVIEW

**Review by royalfig — APPROVED** [review]

File: docs/docs/experiments.mdx [hunk 1]
```diff
@@ -54,7 +54,7 @@ available until your tracking software has loaded, and you may need to use a cus
 
 GrowthBook's feature flags can work just as well from the client/browser as it does server side. The feature flag's
 conditional logic ([rules](/features/rules)) controls how a feature should be shown, and if a feature should be shown as part
-of an experiment. The same targeting and override rules apply exactly the same to client-side experimentation.
+of an experiment. The same  and override rules apply exactly the same to client-side experimentation.
```
Comment (lukesonnet) [inline docs/docs/experiments.mdx:57]: Typo here?

**Review by lukesonnet — CHANGES_REQUESTED** [review]

File: docs/docs/experiments.mdx (hunk 1 above)
Comment (mattdp) [reply docs/docs/experiments.mdx]: fixed thanks!

File: docs/docs/experiments.mdx [hunk 2]
```diff
@@ -47,14 +47,14 @@ handle all of this for you, and you can also serve client-side A/B test via our
 Client-side A/B tests are great for testing visual changes to your application, but they do have some drawbacks. One of
 the most common issues is caused by the delay in loading the specific variation to a user, which may cause a flash or
 flickering as the experiment loads. This can be reduced by using inline-experiments, or by moving the GrowthBook SDK
-code higher in the code so it loads at the same time or before the page. Sometimes the hashing attribute may not be
+code higher in the code so it loads at time or before the page. Sometimes the hashing attribute may not be
```
Comment (lukesonnet) [inline docs/docs/experiments.mdx:50]: you introduced another typo?

**Review by lukesonnet — CHANGES_REQUESTED** [review]

File: docs/docs/experiments.mdx (hunk 2 above)
Comment (mattdp) [reply docs/docs/experiments.mdx:50]: i'm baffled and am open to having made this mistake but really don't think i changed anything here

Comment (mattdp) [reply docs/docs/experiments.mdx:50]: someone with better git-fu please fix and commit, i have limited permissions

**Review by lukesonnet — APPROVED** [review]
sorry to be a stickler and not just resolve it myself but i had limited context.

**Pull request merged**